pkg/cloudprovider/providers/gce/gce_addresses.go

```diff
@@ -82,6 +84,16 @@ func (gce *GCECloud) ReserveAlphaRegionAddress(addr *computealpha.Address, regio
 	return gce.waitForRegionOp(op, region, mc)
 }
 
+// ReserveBetaRegionAddress creates a beta region address
```

Comment:
Curious why the Beta keyword is in the middle? Shouldn't it be a prefix or suffix?